apps/location_service/lib/location_service.ex

```diff
@@ -34,5 +34,13 @@ defmodule LocationService do
     end)
   end
 
-  # TODO Add suggestion/place lookup
+  @doc "Uses either AWS Location Service or Google Maps Place API to do
+  autocompletion, selecting based on config value."
+  @spec autocomplete(String.t(), number) :: LocationService.Suggestion.result()
+  def autocomplete(search, limit) do
```

Comment:
With respect to our earlier chat - this seems to work fine for me, when wrapped in the `cache` function like so

```elixir
cache({search, limit}, fn {search, limit} ->
    # case do blah blah
end)

```

Reply:
This should be good now 👍